packages/toast/src/styles.ts

```diff
@@ -0,0 +1,545 @@
+import { css } from '@leafygreen-ui/emotion';
+import CheckmarkWithCircleIcon from '@leafygreen-ui/icon/dist/CheckmarkWithCircle';
+import InfoWithCircle from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import ImportantWithCircleIcon from '@leafygreen-ui/icon/dist/ImportantWithCircle';
+import RefreshIcon from '@leafygreen-ui/icon/dist/Refresh';
+import WarningIcon from '@leafygreen-ui/icon/dist/Warning';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies, typeScales, spacing } from '@leafygreen-ui/tokens';
+import { transparentize } from 'polished';
+import { Transition } from 'react-transition-group';
+import { StyledElements, Variant } from './types';
+
+export const toastWidth = 400;
+export const transitionDuration = 150;
+
+export const baseElementStyles: Partial<Record<StyledElements, string>> = {
+  toast: css`
+    font-family: ${fontFamilies.default};
+    font-size: ${typeScales.body1.fontSize}px;
+    line-height: ${typeScales.body1.lineHeight}px;
+    position: fixed;
+    bottom: ${spacing[6]}px;
+    left: ${spacing[4]}px;
+    width: ${toastWidth}px;
+    max-width: calc(100vw - ${spacing[4] * 2}px);
+    border-radius: 12px;
+
+    overflow: hidden;
+    transform: translate3d(0, ${spacing[3]}px, 0) scale(0.95);
+    transform-origin: bottom center;
+    opacity: 0;
+    transition: all ${transitionDuration}ms ease-in-out;
+    border: 1px solid;
+
+    a {
+      font-size: inherit;
+      line-height: inherit;
+      font-weight: 700;
+      text-decoration: underline;
+      text-underline-offset: 3px;
+      text-decoration-thickness: 2px;
+      border-radius: 4px;
+      &:hover,
+      &:focus,
+      &:focus-visible {
+        outline: none;
+        span {
+          &::after {
+            display: none;
+          }
+        }
+      }
+      &:focus-visible {
+        position: relative;
+      }
+    }
```

Comment:
Similar to Banner, this may be overridden by some apps' global styles. Is there a way to add more specificity to this?

Reply:
Since this is targeting any `a` inside of the component wouldn't that take precedence over a global `a`? 

```
a {
  color: red;
}

div a {
  color: green;
}
```

Reply:
It should yeah

Reply:
Update looks good 👍